.github/workflows/ci.yml

```diff
@@ -23,21 +23,16 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
-
-      - name: Cache Python dependencies
-        uses: actions/cache@v4
-        id: pip-cache
-        with:
-          path: ~/.cache/pip
-          key: pip-${{ matrix.python-version }}-${{ hashFiles('**/setup.py') }}
-          restore-keys: |
-            pip-${{ matrix.python-version }}-${{ hashFiles('**/setup.py') }}
+          cache: 'pip'
+          cache-dependency-path: setup.py
```

Comment:
CI was failing, so I changed the method of caching.
https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#caching-packages

I have confirmed that the cache is being used.
https://github.com/KitwareMedical/dicom-anonymizer/actions/runs/9184333043/job/25256468458

![Screenshot 2024-05-22 at 11 21 19](https://github.com/KitwareMedical/dicom-anonymizer/assets/41093742/4314a393-5292-484f-84c3-905a2113e66f)